docs/source/configuration/telemetry/instrumentation/selectors.mdx

```diff
@@ -96,6 +96,24 @@ The subgraph service executes multiple times during query execution, with each e
 | `error`                     | No          | `reason`         | A string value containing error reason when it's a critical error              |
 | `cache`                     | No          | `hit`\|`miss`    | Returns the number of cache hit or miss for this subgraph request              |
 
+### Connector
+
+#### HTTP
+
+Apollo Connectors for REST APIs make HTTP calls to the upstream HTTP API. These selectors let you extract metrics from these HTTP requests and responses.
+
+| Selector                         | Defaultable | Values           | Description                                                       |
+|----------------------------------|-------------|------------------|-------------------------------------------------------------------|
+| `subgraph_name`                  | No          | `true`\|`false`  | The name of the subgraph containing the connector                 |
+| `connector_source `              | No          | `name`           | The name of the `@source` associated with this connector, if any  |
+| `connector_http_request_header`  | Yes         | `true`\|`false`  | The name of a connector request header                            |
```

Comment:
Are you sure about the values column here ? Is it really true or false and not the header name ? A string ?